pkg/template/parse.go

```diff
@@ -104,6 +105,9 @@ func (t *Template) parseContent(answersCB AnswerCallback) ([]byte, error) {
 	if err != nil {
 		return nil, err
 	}
+	if template == nil {
```

Comment:
why need this error? We have to be carefully not throwing error when template is invalid. We should just fall back and continue.

Reply:
This will break if user defines the following Riofile:
```yaml
services:
  demo:
    image: foo
    template: true
```

Reply:
That actually works for me with changes, I tested a few different files to see if I could cause error on good Riofile and couldn't. But I'll just have it return content to be safe.